gazebo/igvc_description/urdf/Jaymii.urdf

```diff
@@ -392,63 +392,36 @@
   </transmission>
 
 
-  <joint name="base_footprint_to_base_link" type="fixed">
-    <parent link="base_footprint"/>
-    <child link="base_link"/>
-    <origin xyz="0 0 0.12" rpy="0 0 0"/>
+  <joint name="imu_to_base_footprint" type="fixed">
+    <parent link="imu"/>
+    <child link="base_footprint"/>
+    <origin xyz="0 0 -.633" rpy="0 0 0"/>
     <axis xyz="0 0 0"/>
   </joint>
-  <joint name="base_footprint_to_body" type="fixed">
-    <origin xyz="0 0 0.12" rpy="0 0 0" />
-    <parent link="base_footprint"/>
-    <child link="body"/>
-  </joint>
-  <joint name="base_footprint_to_lidar" type="fixed">
-    <origin xyz="0.4572 0 0.9525" rpy="0 0 0" />
-    <parent link="base_footprint"/>
+  <joint name="imu_to_lidar" type="fixed">
+    <origin xyz=".276 -.029 .029" rpy="0 0 0" />
+    <parent link="imu"/>
     <child link="lidar"/>
   </joint>
-  <joint name="base_footprint_to_imu" type="fixed">
-    <origin xyz="0.0 0 0.9525" rpy="0 0 3.141" />
-    <parent link="base_footprint"/>
-    <child link="imu"/>
-  </joint>
-  <joint name="base_footprint_to_gps" type="fixed">
-    <origin xyz="-0.04572 0.2032 0.9525" rpy="0 0 0" /> <!-- -1.5708 -->
-    <parent link="base_footprint"/>
-    <child link="gps"/>
-  </joint>
-  <joint name="base_footprint_to_center_camera" type="fixed">
-    <origin xyz="0.03302 0 1.7145" rpy="0 0 0" />
-    <parent link="base_footprint"/>
+  <joint name="imu_to_center_camera" type="fixed">
+    <origin xyz="-.056 0 .937" rpy="0 1.03 0" />
+    <parent link="imu"/>
     <child link="usb_cam_center"/>
   </joint>
-  <joint name="body_to_left_camera" type="fixed">
-    <origin xyz="0.0 0.4 0.4" rpy="0 0.174533 0.785398" />
-    <parent link="body"/>
-    <child link="usb_cam_left"/>
-  </joint>
-  <joint name="body_to_right_camera" type="fixed">
-    <origin xyz="0.0 -0.4 0.4" rpy="0 0.174533 -0.785398" />
-    <parent link="body"/>
-    <child link="usb_cam_right"/>
-  </joint>
-  <joint name="base_footprint_to_axle" type="fixed">
```

Comment:
As is this does not have a parent so you technically have two... Since it is used in gazebo things we technically cannot remove it. I will give more detailed thoughts next meeting. Always good to run 
`check_urdf FILE_NAME.urdf`